### ISSUE TITLE
option "Unfocused auto-scroll" doesn't work on release Version 1.4.5 (TD 3.1.1) Latest

### ISSUE THREAD

**Issue by d00m178** [kotatogram/kotatogram-desktop#263]
### Steps to reproduce

- configure option "Unfocused auto-scroll" to Enabled state in kotogram app
- open some group chat in app
- unfocus app window

### Expected behaviour

new messages in chat should auto-scrolling to the last message

### Actual behaviour

after about 10 new messages auto-scrolling stop working (like in the original telegram desktop app)

### Operating system

win10

### Version of Kotatogram Desktop

Version 1.4.5 (TD 3.1.1) Latest portable

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
Hmm, I can't reproduce, I'm on Linux though. Are you sure the window is not overlapped by anything?

**Comment by d00m178** [comment]
да, окно было перекрыто другим окном. оба они - на другом мониторе. 
я думал, что если окно котограмма не в фокусе, то все равно - перекрыто оно или нет.
я не прав? окно должно быть поверх всех окон и только тогда будет работать скроллинг?
тогда не очень ясен смысл этой опции.. хочется же оставить кусочек окна на экране, в котором будут пролистываться сообщения, всегда отображая последнее..

**Comment by ilya-fedin** [comment]
Не, там есть проверка, что окно не перекрыто, чтобы не происходило такого, что оно где-то на заднем плане, а оно все равно там листается и непонятно, где ты остановился

**Comment by d00m178** [comment]
сейчас проверил - действительно, если все окно котограмма целиком видно на экране, то если переключиться с него на другое (на другом мониторе) - скроллинг работает как надо, все листается вниз автоматом..
но только открыть окно (например хром), которое будет НЕМНОГО скраю перекрывать котограм - все, вижу, что 10 сообщений и скролинг перестает работать.
как же так? оно же по прежнему НЕ в фокусе, я даже не трогал его. но мне прекрасно видно область чата и мне бы хотелось в таком виде и наблюдать за чатом

**Comment by d00m178** [comment]
> Не, там есть проверка, что окно не перекрыто, чтобы не происходило такого, что оно где-то на заднем плане, а оно все равно там листается и непонятно, где ты остановился

вас понял... возможно есть смысл добавить чекбокс в настройках, для отключения такой проверки..
кейс я описал вроде) ну вам решать. 
спасибо за объяснение.

**Comment by ilya-fedin** [comment]
ну, если туда добавлять чекбоксы по каждой мелочи, то опций будет столько, что никто не в силах будет их все просмотреть :(

**Comment by hunterfrerich** [comment]
> well, if you add checkboxes for every little thing there, then there will be so many options that no one will be able to view them all :(  

I downloaded this fork of telegram specifically to auto-scroll when telegram is completely hidden by other windows. I use OBS to window-capture my Telegram message stream, to record or display it elsewhere. Windows are still accessible to OBS for recording if they are bottom-most (but not it they are dragged off-screen). I notice you said there would be too many checkboxes if you added this feature, but I think many people have similar use-cases.
  
I tried another telegram client, Unigram, and noticed that it does auto-scroll when behind windows. You just need to uncheck "count unread messages" in settings. I would prefer to use Kotatogram, but until this feature is considered, I'll use Unigram.

**Comment by ilya-fedin** [comment]
> I use OBS to window-capture my Telegram message stream, to record or display it elsewhere.

What's the point for this?

**Comment by hunterfrerich** [comment]
> What's the point for this?  
  
Integration/overlay onto camera feed:  
https://randomnerdtutorials.com/esp8266-nodemcu-door-status-telegram/  
![example-camera-overlay](https://i.imgur.com/b6XsSnQ.png)   
  
This way is easier than coding something by hand with the API.

**Comment by ilya-fedin** [comment]
Easier than using e.g. telethon?

**Comment by hunterfrerich** [comment]
> telethon  
  
Oooh python, nice. Thank you for that, and i'll look into it. I could still imagine some non-technical users might want to capture a message stream, but I understand you have to balance the features... Still though, Unigram seems to have figured out the right tradeoff. They only have one checkbox to do it, yet still show unread message count when it's first opened.  
  
Either way, thanks for your work.
